source/_integrations/scrape.markdown

```diff
@@ -21,7 +23,15 @@ If you are not using Home Assistant Container or Home Assistant Operating System
 sudo apt install libxml2
 ```
 
-To enable this sensor, add the following lines to your `configuration.yaml` file:
+Check Beautifulsoup's [CSS selectors](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#css-selectors) for details on how to write a **Select**.
+
+You can enable this integration using both from the user interface or by adding yaml to your `configuration.yaml` file.
```

Comment:
There is a grammatical error here and it can probably be shortened as almost the same is written later on.